MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -1015,6 +1015,7 @@ protected override string GetCommandLineArguments(string outputPath, EncodingOpt
 
             var mapArgs = state.IsOutputVideo ? EncodingHelper.GetMapArgs(state) : string.Empty;
 
+            // TODO: Path.GetDirectoryName could return null
             var outputTsArg = Path.Combine(Path.GetDirectoryName(outputPath), Path.GetFileNameWithoutExtension(outputPath)) + "%d" + GetSegmentFileExtension(state.Request);
```

Comment:
Again, only if outputPath is null or a root dir

Reply:
done